saltstack/base/salt/prerequisites/usr/bin/user-data-helper.sh

```diff
@@ -1,7 +1,13 @@
 #!/bin/bash
 
 set -x
-set
+
+if [[ "$SECRET_ENCRYPTION_ENABLED" == "true" ]]; then
+  set | /usr/bin/redact-secrets.sh
+  source /usr/bin/retrieve-userdata-secrets.sh $> /var/log/retrieve-userdata-secrets.log
```

Comment:
Nit: For consistency, perhaps add `cdp-` prefix to `retrieve-userdata-secrets.sh` to make it in sync with `cdp-luks`. Same goes for `retrieve-userdata-secrets.log`.